### ISSUE TITLE
Accessibility: Object Status: Different numeric object statuses are read together as a single number at first tab stop, when accessed. 

### ISSUE THREAD

**Issue by puru-hk** [SAP/fundamental-ngx#3449]
Description: Different numeric object statuses are read together as a single number at first tab stop, when accessed. 

Expected: All object statuses should not be read continuously. different object statuses should be read separately

Screenshot: 

![image](https://user-images.githubusercontent.com/32538291/94521410-a0e38480-024b-11eb-8ffd-af264001af69.png)

![image](https://user-images.githubusercontent.com/32538291/94521421-a4770b80-024b-11eb-9731-c6527d6954bc.png)

**Comment by Lokanathan-k** [comment]
fixed as part of the PR https://github.com/SAP/fundamental-ngx/pull/4176
